Makefile

```diff
@@ -44,6 +44,13 @@ $(OUT_DIR)/$(BIN)-%:
 
 build: $(OUT_DIR)/$(BIN)
 
+.PHONY: update-go-deps
+update-go-deps:
+	@for m in $$(go list -mod=readonly -m -f '{{ if and (not .Indirect) (not .Main)}}{{.Path}}{{end}}' all); do \
+		go get $$m; \
```

Comment:
i believe we want to `go get -d`, see https://github.com/prometheus/prometheus/blob/b48a1e9e9bc3575e2f6ef3e7ea8573434288e0bb/Makefile.common#L159-L168

according to `get help get`:
```
The -d flag instructs get not to build or install packages. get will only
update go.mod and download source code needed to build packages.
```